packages/gatsby-plugin-image/src/components/main-image.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { forwardRef } from "react"
+import { Picture, PictureProps } from "./picture"
+
+export type MainImageProps = PictureProps
+
+export const MainImage = forwardRef<HTMLImageElement, MainImageProps>(
+  function MainImage({ ...props }, ref) {
+    return <Picture ref={ref} {...props} />
+  }
+)
+
+MainImage.displayName = `MainImage`
+MainImage.propTypes = Picture.propTypes
```

Comment:
Do we still need this if we're typing?

Reply:
I'm not really sure about propTypes. I never use them myself, but they are technically solving a slightly different problem than TS types, as they're evaluated at runtime.